lib/security/WSSecurityCert.js

```diff
@@ -44,8 +44,8 @@ function WSSecurityCert(privatePEM, publicP12PEM, password, options) {
     passphrase: password
   };
   this.x509Id = "x509-" + generateId();
-  this.hasTimeStamp = options.hasTimeStamp || typeof options.hasTimeStamp === 'boolean' ? !!options.hasTimeStamp : true;
-  this.signatureTransformations = options.signatureTransformations && Array.isArray(options.signatureTransformations) ? options.signatureTransformations
+  this.hasTimeStamp = typeof options.hasTimeStamp === 'undefined' ? true : !!options.hasTimestamp;
```

Comment:
I'm sorry to be a bit picky here, but I think we should stuck with either `hasTimeStamp` or `hasTimestamp` in order to have it consistent between the `options` and the internal value. I'd vote for `hasTimestamp` as I haven't seen the other version too often, but it's up to you as long as it's the same 😉 👍

Reply:
I only use hasTimeStamp because WSSecurity uses this, I actually prefer hasTimestamp, because de xml field has the same casing.

And the inconsistency I just missed. The typeof field has casing. It got through the test because !!undefined === false. The code works as expected.

It is ok to have different field names for the same functionality in different classes? If yes, I happily change it to hasTimestamp.

Reply:
Ah, I see. No, just leave it that way (`hasTimeStamp`) but make it consistent in the `options` too, so that it doesn't get too confusing ;)